.github/workflows/build.yml

```diff
@@ -1,13 +1,15 @@
-name: Building
+name: Build Torus Runtime
 
 on:
   workflow_dispatch:
   push:
-    branches: [github-ci-test]
-
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
```

Comment:
why take this out?

Reply:
![image](https://github.com/user-attachments/assets/5ff3ff2c-26b6-4ffd-852d-da60aacf7388)

It seems it was either you, or someone(me) fucking a rebase lol

Reply:
Also, I think we should allow merge instead of squash in some cases. This PR is one of them. I can explain later.